public/js/spec/articleSpec.js

```diff
@@ -0,0 +1,24 @@
+let article;
+
+describe('Article', () => {
+  describe('headline', () => {
+    it('shows the article headline', () => {
+      article = new Article({headline: "Test"});
```

Comment:
so i had read the project challenge, in saying we weren't to use any backend javascript, to mean that we shouldn't write any classes or anything and that everything should be done from the interface? i might have got the wrong end of the stick there tho.

Reply:
Hey Joey! No, in this case it means that you shouldn't write any code that runs on a server rather than in the browser. It's still a good idea to write your code with good separation of concerns and separate your interface from your models even when writing code for the browser. You can think of backend and frontend as synonyms for server and client.